### DISCUSSION TITLE
java-agent pulgin support dubbo2.6.x?

### DISCUSSION THREAD

**Discussion by hu-jiawen-git** [apache/skywalking#9746] [Q&A] [answered]
java-agent pulgin support dubbo2.6.x?

**Comment by hu-jiawen-git** [comment, marked as answer]
in the https://skywalking.apache.org/docs/skywalking-java/v8.10.0/en/setup/service-agent/java-agent/supported-list/
support dubbo 2.5.4 -> 2.6.0.
but test the 2.6.2. supported. 
the docs's wrong?

**Reply by wu-sheng** [reply]
The versions in the doc usually mean those passed automatical testing through CI.
This doesn't mean others are not. We are not able to follow hundreds of communities iteration.

**Reply by hu-jiawen-git** [reply]
It means Whether dubbo 2.6. x supports depends on the test results.
